content/en/docs/Concepts/olm-architecture/dependency-resolution/_index.md

```diff
@@ -75,13 +75,17 @@ properties:
 
 This structure can be used to construct a CEL expression for generic constraint. See [Generic Constraint](#generic-constraint) for more information.
 
-Note: `properties.yaml` is supported in `0.17.4+` versions of OLM.
+Note: `properties.yaml` is supported in `1.17.4+` versions of OPM.
 
 ## Declaring Dependencies
 
-Dependencies are declared by including a `dependencies.yaml` file in the `metadata` directory of the operator bundle. For more information on bundles and the their format, see the [bundle docs](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md).
+Required dependencies are statically defined in the operator bundle one of two ways: 
+- `dependencies.yaml` file
+- `properties.yaml` file (supported by `1.17.4+ versions of OPM)
+ 
+Each of these files can be included in the `metadata` directory of the operator bundle. For more information on bundles and the their format, see the [bundle docs](https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md).
```

Comment:
would be nice at some point we have this doc on the olm website.

Reply:
I assume once this is merged, it will show up here:
https://olm.operatorframework.io/docs/concepts/olm-architecture/dependency-resolution/

Isn't this the "OLM website"?

Reply:
I don't believe we have documentation around the OLM bundle format (e.g. registry+v1) on the official OLM documentation site, so we always reference the documentation that lives in the operator-registry repository as the source of truth. I think Camila was alluding to mirroring that operator-bundle.md documentation to the olm.operatorframework.io website, so we can have all documentation under one roof, vs. scattered throughout multiple repositories.

Reply:
Probably tie into this:
https://github.com/operator-framework/olm-docs/issues/187